package.json

```diff
@@ -49,7 +49,6 @@
     "@springernature/frontend-package-manager": "^6.1.3",
     "@springernature/sasslint-config": "^1.2.1",
     "@springernature/util-eslint-runner": "^0.0.3",
-    "@springernature/util-sass-compiler": "^0.1.0",
```

Comment:
Can we also leave this and I am actually going to be using it in an upcoming PR!